play-json/shared/src/main/scala-3/play/api/libs/json/ImplicitResolver.scala

```diff
@@ -0,0 +1,263 @@
+/*
+ * Copyright (C) 2009-2021 Lightbend Inc. <https://www.lightbend.com>
```

Comment:
I think this shouldn't be used anymore, please verify with someone that is more aware of these details

Reply:
All the existing files have this, including the Scala2 version of `ImplicitResolver` on which this is based. Until it's removed everywhere, better to have it there.